stories/Checkbox.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Checkbox as Component  } from "../confetti-ds/src";
+
+export default {
+  title: "Checkbox/Checkbox",
+  component: Component,
+  argTypes: {
+    value: {
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
+      }
+    }
+  }
+};
+
+export const Checkbox = (args) => (
+  <fieldset>
+    <Component
+      id="demo-checkbox-1"
+      name="demo-checkbox"
+      label="checkbox 1"
```

Comment:
same thing as in radio: the storybook controls aren't applying to the components, since you're not passing them as args.

Reply:
fixed